conductor-client/src/main/kotlin/com/openlattice/data/jobs/DataDeletionJob.kt

```diff
@@ -286,36 +327,50 @@ class DataDeletionJob(
         ps.addBatch()
     }
 
+    //TODO: entity key ids should be paired with their entity key ids :-/
```

Comment:
They should be paired with themselves? Should this be "paired with their entity **set** ids"?